### ISSUE TITLE
Looks like resolrize has been failing

### ISSUE THREAD

**Issue by carolyncole** [psu-libraries/scholarsphere-3#684]
This may be the reason for #416

```
job_class: ResolrizeJob
job_id: ecf9405c-68e2-4c33-af7a-7d9fc640de95
queue_name: default
arguments: []
locale: en
Exception
NoMethodError
Error
undefined method `rdf_source?' for #<Ldp::NoneClass:0x007f49cadf89a8>
```

**Comment by carolyncole** [comment]
There was a fix put in ScholarSphere 2.0 to index based on id length and then there was a change in Sufia 7 to re-index for members. https://github.com/psu-stewardship/scholarsphere/blob/master/app/jobs/resolrize_job.rb

**Comment by carolyncole** [comment]
Model ids are now 10 digits long instead of 9
